explorer/app/components/Project/components/Contacts/contacts.tsx

```diff
@@ -24,23 +23,21 @@ interface Props {
 export const Contacts = ({ contacts }: Props): JSX.Element => {
   return (
     <Section collapsable title="Contact">
-      <SectionContent gap={4}>
```

Comment:
SectionContent has been relocated to inside `Section` here and in other project detail components.